src/main/java/miles/task/Task.java

```diff
@@ -0,0 +1,104 @@
+package miles.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/*
+ * Represents a general task.
+ */
+public abstract class Task {
+    private boolean isDone;
+    private String task;
+
+    public Task(String task) {
+        this.isDone = false;
+        this.task = task;
+    }
+
+    /**
+     * Marks the current task as done.
+     */
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    /**
+     * Marks the current task as undone.
+     */
+    public void markAsUndone() {
+        this.isDone = false;
+    }
+
+    /**
+     * Checks if a task has no description by checking if the strings are equal.
+     * 
+     * @param taskType  the type of task (e.g. "deadline")
+     * @param task      the command inputted by the user
+     * @return          a boolean that indicates whether the task has no description
```

Comment:
Its a little confusing what the boolean value means 
```suggestion
     * @return          true if the task has no description
```